complex_src.c

```diff
@@ -0,0 +1,62 @@
+#include<stdio.h>
+#include "complex_header.h"
+
+/**
+ * @brief add function of structure type returns real and imaginary part
+ * 
+ * @param first 
+ * @param second 
+ * @return complex_t 
+ */
+complex_t add(complex_t first, complex_t second)
+{
+    complex_t temp;
+    temp.real = first.real + second.real;
+    temp.img = first.img + second.img;
+    return(temp);
+}
+
+/**
+ * @brief subtract function of structure type returns real and imaginary part

```

Comment:
No need to add doxygen comments in both .c and .h. Doxygen will duplicate if both files are used for documentation.